src/ape_test/providers.py

```diff
@@ -76,6 +76,11 @@ def revert(self, snapshot_id: SnapshotID):
     def set_timestamp(self, new_timestamp: int):
         self._tester.time_travel(new_timestamp)
 
+    def mine(self, num_blocks: int):
+        for i in range(num_blocks):
+            txn = TransactionAPI()
+            self.send_transaction(txn)
```

Comment:
So `eth-tester` has this functionality already:
https://github.com/ethereum/eth-tester/blob/master/eth_tester/main.py#L348

```suggestion
        self._tester.mine_blocks(num_blocks)
```

Reply:
correction made